app/dataController/projects.controller.ts

```diff
@@ -26,15 +26,36 @@ export interface IArchiveProjects {
   projectId: number
   status: 'Active' | 'Archived' | 'Deleted'
   userId: number
+  projectName?: string
 }
 
 const ProjectsController = {
   getAllProjects: (params: IGetAllProjects) => ProjectsDao.getAll(params),
   getProjectInfo: (projectId: number) => ProjectsDao.getProjectInfo(projectId),
   createProject: (params: ICreateProject) => ProjectsDao.createProject(params),
   editProject: (params: IEditProject) => ProjectsDao.editProject(params),
-  updateProjectStatus: (params: IArchiveProjects) =>
-    ProjectsDao.updateProjectStatus(params),
+  updateProjectStatus: async (params: IArchiveProjects) => {
+    const {projectId, status} = params
+    if (status === 'Archived') {
```

Comment:
```suggestion
    if (status === 'Archived' || status === 'Deleted') {
```
let's handle for both the cases